fenixedu-ist-integration/src/main/java/pt/ist/fenixedu/integration/api/FenixAPIv1.java

```diff
@@ -1078,6 +1081,46 @@ public String enrolledByDegreeAndCurricularYear(@PathParam("id") String degreeId
         }).collect(StreamUtils.toJsonArray()).toString();
     }
 
+    @GET
+    @Produces(JSON_UTF8)
+    @Path("users")
+    @OAuthEndpoint(value = PERSONAL_SCOPE, serviceOnly = true)
+    public String getUsers() {
+        Stream<User> users = Bennu.getInstance().getUserSet().stream();
```

Comment:
I understand that. My reasoning here is that we can all agree that the `JerseyServices` implementation is far from perfect.

If we're adding the "proper" API we've been discussing for a while, it should be the better version, not preserving the "martelanço" we kept here. For the time being, I think that peripheral applications should use the old API, until the new one is finished.